packages/apps/plugins/plugin-markdown/src/components/StandaloneLayout.tsx

```diff
@@ -24,8 +24,8 @@ export const StandaloneLayout = ({
         <div
           role='none'
           className={mx(
-            inputSurface,
-            surfaceElevation({ elevation: 'group' }),
+            // inputSurface,
+            // surfaceElevation({ elevation: 'group' }),
```

Comment:
Rich and I added these to harmonize MarkdownPlugin’s main surface with other plugins’ main surfaces. I think making experiences that are equivalent also look equivalent and look like each other is a worthwhile design hypothesis. I'd prefer to continue exposing folks to that idea before going back to a treatment just for MarkdownPlugin that is flat & transparent like the other static elements in the UI.

Reply:
not sure which other plugins we should care about here, but i'm pretty convinced we don't want another frame within-a-frame when we're working on a document, the rest of the page furniture already creates a nice margin around the text, the extra lines add noise imo. I'd push for not having any elevation here, what other patterns are offended by this?